clevertap-core/src/main/java/com/clevertap/android/sdk/ControllerManager.java

```diff
@@ -218,4 +220,11 @@ public void invokeCallbacksForNetworkError() {
 
         // Add more callbacks if necessary
     }
+
+    public void invokeBatchListener(JSONArray requestQueue, boolean success) {
```

Comment:
add documentation here, mention is this invoked pre api call or post api call

Reply:
added changes in https://github.com/CleverTap/clevertap-android-sdk/commit/1c00a7717771993d2bc5bbdd33b8b797081e129e